Userland/Libraries/LibWeb/XLink/AttributeNames.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2023, Shannon Booth <shannon@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Error.h>
+#include <AK/FlyString.h>
+
+namespace Web::XLink::AttributeNames {
+
+#define ENUMERATE_XLINK_ATTRIBUTES(E) \
+    E(type)                           \
+    E(href)                           \
+    E(role)                           \
+    E(arcrole)                        \
+    E(title)                          \
+    E(show)                           \
+    E(actuate)                        \
+    E(label)                          \
+    E(from)                           \
+    E(to)
```

Comment:
This not being alphabetical bothers me a bit but I'm going to merge this anyway to save you from any further conflicts!